src/subscriber/subscriberV2.ts

```diff
@@ -94,6 +109,18 @@ export default class SubscriberV2 extends Subscriber {
         };
 
         return class extends subscriberClass {
+          constructor(...args: any[]) {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8095345)
```suggestion
          constructor(...args: unknown[]) {
```